src/components/project/project-step.resolver.ts

```diff
@@ -26,6 +34,22 @@ export class ProjectStepResolver {
     );
   }
 
+  @ResolveField(() => [ProjectStepChange], {
+    description: 'The history of project steps.',
+  })
+  async history(
+    @Parent() step: SecuredProjectStep & { parentId: ID; changeset?: ID },
+    @AnonSession() _session: Session
+  ): Promise<ProjectStepChange[]> {
+    if (!step.canRead || !step.canEdit || !step.value) {
```

Comment:
```suggestion
    if (!step.canRead || !step.value) {
```
You don't need edit permission to view the list